booster_bdd/config/prod-preview.sh

```diff
@@ -18,6 +18,9 @@ export WIT_API="${WIT_API:-https://api.prod-preview.openshift.io}"
 ## URI of the Openshift.io's Auth server
 export AUTH_API="${AUTH_API:-https://auth.prod-preview.openshift.io}"
 
+## URI of the Openshift.io's Che server
+export CHE_API="${CHE_API:-https://che.openshift.io}"
+
```

Comment:
The following variable used in `config/config.sh` is missing here:
```
## github repo URL (for use in creating Che workspace)
export GIT_REPO_URL="${GIT_REPO:-https://github.com/che-samples/console-java-simple.git}"

```

Reply:
Change made - thanks!